src/screens/repo/screens/build/menu.js

```diff
@@ -79,6 +114,26 @@ export default class BuildMenu extends Component {
 									</button>
 								)}
 							</li>
+							<li>
+								{build.status === "success" ? (
+									<button onClick={this.togglePromote}>
+										<DeployIcon />
+										<span>Promote Build</span>
+									</button>
+								):null }
+							</li>
+							{build.status === "success" && this.state.togglePromote ? (
+								<ul class="sub">{envs.map(function(env) {
```

Comment:
Since this item belong to previous section "Promote" maybe we should move it inside the `<li>` 
 tag?